test/unit/variable.test.js

```diff
@@ -166,6 +166,18 @@ describe('Variable', function () {
         expect(v1.value).to.equal('{"foo":"bar"}');
     });
 
+    it('should set value to null when value provided as cyclic object value (json)', function () {
+        var obj = {};
+        obj.prop = obj;
+        // eslint-disable-next-line one-var
```

Comment:
why?

Reply:
Can't define two variables separately, need to initialize `obj` first before using.